### PR TITLE
Front-end env setup

### PR DESCRIPTION
**Pull request by sachabinder**
Setup of the font-end dockerized.
* Framework : ReactJS + Sass
* Bundler : webpack
* Transpiler : babel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +98,8 @@ To manage dependancy on the backend, we are using [poetry](https://python-poetry
 | DB_USER | User of the DB | uquizz |
 | DB_NAME | Name of the DB | uquizz |
 | DB_PASSWORD | Password of the DB | uquizz |
-| BACK_PORT | External port back access | 8000 |
+| BACK_PORT | External back port access | 8000 |
+| BACK_PORT | External front port access | 8080 |
```
Comment (StephaneGrabsia) [inline README.md]: A little typo here, I think you meant FRONT_PORT ;)

File: back/Dockerfile [hunk 2]
```diff
@@ -74,10 +74,11 @@ RUN apt-get update \
     netcat=1.10-46 \
     && rm -rf /var/lib/apt/lists/*
 
-COPY ./entrypoint.dev.sh .
-RUN chmod +x /src/entrypoint.dev.sh
+COPY ./entrypoint.dev.sh /usr/local/bin/
```
Comment (StephaneGrabsia) [inline back/Dockerfile:77]: I don't understand why you didn't copy the entrypoint.dev.sh in the same folder than the entrypoint.prod.sh

**Review by StephaneGrabsia — APPROVED** [review]
CR OK, everything looks fine. It's working on my computer. Good job! (We were this close to succeed last week with the Dockerfile!)

File: README.md (hunk 1 above)
Comment (sachabinder) [reply README.md]: right !

File: back/Dockerfile (hunk 2 above)
Comment (sachabinder) [reply back/Dockerfile:77]: because I am stupid, I forgot to change that... :smile:

**Pull request merged**